katana/framework.py

```diff
@@ -91,22 +116,55 @@ def __init__(self, fp_dem, zone_letter, zone_number, buff, start_date,
         self.ts = get_topo_stats(self.fp_dem)
         self.x1 = self.ts['x']
         self.y1 = self.ts['y']
-        # WindNinja grid spacing
-        self.dxy = dxy
 
-    def create_log(self, loglevel, logfile):
+    def parse_config(self):
+        """Parse the config file variables for running katana
+
+        Raises:
+            None
+        """
+
+        # topo section
+        self.fp_dem = self.config['topo']['filename']
```

Comment:
Could consider keeping self attribute naming convention consistent with config option (self.filename)

Reply:
I left most of the variable names alone so I knew they would work instead of changing them all. This will all be gone over again with #8 to support WRF files.